torch/multiprocessing/queue.py

```diff
@@ -24,6 +24,12 @@ def recv(self):
     def __getattr__(self, name):
         return getattr(self.conn, name)
 
+    def __setstate__(self, state):
+        self.__dict__ = state
+
+    def __getstate__(self):
+        return self.__dict__
```

Comment:
Can you explain what's happening in here? IIRC the default implementations of `__setstate__` and `__getstate__` do exactly this, so I can't see why is this needed

Reply:
I found the original discuss about this [here](https://bugs.python.org/issue5370). So I'll revert the changes.

Reply:
Although the default implementations of `__setstate__` and `__getstate__` do exactly this, they cannot avoid calling `__getattr__` here and need calling `__getattribute__` once or equivalence function or some condition to interrupt.